notebooks/housing_household_recovery.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 33,
```

Comment:
Similar to the comment I left on your other PR, could you remove the output cells?
The conventions is: 
- for the analysis notebooks we do not keep any output
- for testbed notebooks (those under `manual_jb/content/notebooks`) we keep the outputs for display